File: android/src/main/java/com/mobilenativebarcodegenerator/MobileNativeBarcodeGeneratorPackage.java
package com.mobilenativebarcodegenerator;

import com.facebook.react.ReactPackage;
import com.facebook.react.bridge.NativeModule;
import com.facebook.react.bridge.ReactApplicationContext;
import com.facebook.react.uimanager.ViewManager;

import java.util.Collections;
import java.util.List;

/**
 * Lets React Native autolinking discover the Android project. The actual
 * TurboModule implementation is registered and instantiated from C++.
 */
@SuppressWarnings({"deprecation", "rawtypes"})
public final class MobileNativeBarcodeGeneratorPackage implements ReactPackage {
  @Override
  public List<NativeModule> createNativeModules(ReactApplicationContext context) {
    return Collections.emptyList();
  }

  @Override
  public List<ViewManager> createViewManagers(ReactApplicationContext context) {
    return Collections.emptyList();
  }
}

File: android/src/main/java/com/mobilenativebarcodegenerator/MobileNativeBarcodeGeneratorPackage.kt
package com.mobilenativebarcodegenerator

import com.facebook.react.ReactPackage
import com.facebook.react.bridge.NativeModule
import com.facebook.react.bridge.ReactApplicationContext
import com.facebook.react.uimanager.ViewManager


class MobileNativeBarcodeGeneratorPackage : ReactPackage {
  override fun createNativeModules(reactContext: ReactApplicationContext): List<NativeModule> {
    return listOf(MobileNativeBarcodeGeneratorModule(reactContext))
  }

  override fun createViewManagers(reactContext: ReactApplicationContext): List<ViewManager<*, *>> {
    return emptyList()
  }
}
